tests/functional/UseCase/Workspace/CreateDropWorkspaceTest.php

```diff
@@ -233,6 +234,25 @@ public function testCreateDropWorkspace(): void
         );
 
         $this->assertNull($datasets->getIterator()->current());
+
+        $cloudResourceManager = $this->clientManager->getCloudResourceManager($this->projectCredentials);
+        $actualPolicy = $cloudResourceManager->projects->getIamPolicy(
+            'projects/' . $projectId,
+            (new Google_Service_CloudResourceManager_GetIamPolicyRequest()),
```

Comment:
Tady by to mělo byt imho GetIamPolicyRequest ne ten PSR-0 namespace, ale jinak je to asi cajk.